Kernel/Memory/AnonymousVMObject.h

```diff
@@ -83,11 +85,13 @@ class AnonymousVMObject final : public VMObject {
     };
 
     LockWeakPtr<AnonymousVMObject> m_cow_parent;
+    Spinlock m_shared_cow_lock { LockRank::None };
     LockRefPtr<SharedCommittedCowPages> m_shared_committed_cow_pages;
 
     bool m_purgeable { false };
     bool m_volatile { false };
     bool m_was_purged { false };
+    Spinlock m_attributes_lock { LockRank::None };
```

Comment:
Can we use SpinlockProtected here instead (e.g by making a little `struct` to hold the three flags)?

Reply:
Done.